waveshare2in13v2/drawing.go

```diff
@@ -44,54 +44,150 @@ func setMemoryArea(ctrl controller, area image.Rectangle) {
 type drawOpts struct {
 	commands []byte
 	devSize  image.Point
+	origin   Corner
 	buffer   *image1bit.VerticalLSB
 	dstRect  image.Rectangle
 	src      image.Image
 	srcPts   image.Point
 }
 
 type drawSpec struct {
-	// Destination on display in pixels, normalized to fit into actual size.
-	DstRect image.Rectangle
+	// Amount by which buffer contents must be moved to align with the physical
+	// top-left corner of the display.
+	//
+	// TODO: The offset shifts the buffer contents to be aligned such that the
+	// translated position of the physical, on-display (0,0) location is at
+	// a multiple of 8 on the equivalent to the physical X axis. With a bit of
+	// additional work transfers for the TopRight and BottomLeft origins should
+	// not require per-pixel processing by exploiting image1bit.VerticalLSB's
+	// underlying pixel storage format.
+	BufferDstOffset image.Point
```

Comment:
If you don't intent to export the struct, it'd be good to not export its fields.

Reply:
Done.